pkg/controller/networkpolicy/validate_test.go

```diff
@@ -1528,14 +1616,236 @@ func TestValidateAntreaGroup(t *testing.T) {
 					},
 				},
 			},
+			operation:      admv1.Create,
 			expectedReason: "At most one of podSelector, externalEntitySelector, serviceReference, ipBlocks or childGroups can be set for a Group",
 		},
+		{
+			name: "anp-group-set-with-podselector-specified-to-update",
+			curGroup: &crdv1alpha3.Group{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "anp-group-set-with-podselector-specified-to-update",
+					Namespace: "x",
+				},
+				Spec: crdv1alpha3.GroupSpec{
+					PodSelector: &metav1.LabelSelector{
+						MatchLabels: map[string]string{"foo=": "bar"},
```

Comment:
this is an disallowed label key constructed intentionally, I believe other tests use it to fail cases. If the new one passes, there must be something wrong in `validateAntreaGroup`

Reply:
In this case , the old group contains IPBlocks specified  which results in error case `At most one of podSelector, externalEntitySelector, serviceReference, ipBlocks or childGroups can be set for a Group`  so in updating the current group doesn't contain IPBlocks , then its passing .